docs/docs/data/datasets.md

```diff
@@ -154,7 +155,7 @@ INSTRUCTION_DATASETS = {
 #### 4. Stage your changes and run the pre-commit hook
 
 ```bash
-pre-commit run
+pre-commit run --all-files
```

Comment:
You don't need the flag `--all-files` if you run `pre-commit` on staged changes before commit. In this case pre-commit hooks will be applied only to your changes so it allows to avoid redundant long-running hooks, e.g. `npm install` for website folder